README.rst

```diff
@@ -89,8 +227,27 @@ When using the AutoExtract middleware, there are some limitations.
   so it's best to use ``AUTHTHROTTLE_ENABLED=False`` in the settings.
 * Redirects are handled by AutoExtract, not by Scrapy,
   so these kinds of middlewares might have no effect
-* Retries should be disabled, because AutoExtract handles them internally
-  (use ``RETRY_ENABLED=False`` in the settings)
-  There is an exception, if there are too many requests sent in
-  a short amount of time and AutoExtract returns HTTP code 429.
-  For that case it's best to use ``RETRY_HTTP_CODES=[429]``.
+* AutoExtract doesn't handle retries internally, they should be handled by its clients.
```

Comment:
Let's remove this line. It is an implementation detail which can change, and it is not a limitation of scrapy-autoextract, it is an API behavior.

Reply:
Removed.